### PR TITLE
A unit test for minting ERC20 tokens to local accounts on Node 1

### PR DESCRIPTION
**Pull request by vkomenda**

### CODE REVIEW

**Comment by vkomenda** [comment]
If we manage to keep minting and staking in tests rather than split between migration and tests, we can make tests replayable. At the moment the initial balance assertion makes the test fail after the first time.

**Comment by afck** [comment]
That's true, it would be handy for integration testing to have scripts available to e.g. mint more tokens for all validators.
On the other hand it sounds like an abuse of the concept of a "unit test", and should maybe be a separate script instead?

**Comment by vkomenda** [comment]
If tests are replayable then they are more like unit tests rather than integration tests. I agree that for integration tests the ability of being replayed does not matter that much. Integration with `truffle test` is more convenient than a separate script, I think.

File: test/token.js [hunk 1]
```diff
@@ -20,4 +24,18 @@ contract('TestToken', _accounts => {
     let erc20TokenContract = await ValidatorSetContract.instance.methods.erc20TokenContract().call();
     assert.equal(erc20TokenContract.valueOf(), instance.address);
   });
+
+  it('should mint tokens', async () => {
+    assert(accounts.length > 0, 'no accounts to carry on the test');
+    let instance = await Token.deployed();
+    const increment = new BN(1);
```
Comment (varasev) [inline test/token.js]: Are you minting only `1 wei` here?

Comment (vkomenda) [reply test/token.js]: #24 does this better. If it is fixed and merged, this PR can close.

**Pull request closed without merging**